packages/z/zeek/package.yml

```diff
@@ -0,0 +1,46 @@
+name       : zeek
+version    : 6.0.2
```

Comment:
Also, I need a solus-sc PR removing `bro` and related packages from the repo.

Reply:
That PR needs to link to this PR.

Reply:
I told them I would do this part for them 2 months ago and was a slacker. Resolved now.